src/com/google/enterprise/adaptor/sharepoint/SharePointAdaptor.java

```diff
@@ -3036,14 +3042,30 @@ private void getListItemDocContent(Request request, Response response,
         }
         metadataLength += addMetadata(
             response, METADATA_OBJECT_TYPE, ObjectType.LIST_ITEM.value());
+        String contentType = row.getAttribute(OWS_CONTENTTYPE_ATTRIBUTE);
+        if ("Message".equals(contentType)) {       
+        // discussion replies will be of content type "Message".
```

Comment:
Comment wasn't changed. Replace with a one-liner and indent two spaces:

// There is no title field for discussion replies, use DiscussionTitle.